src/components/TodoFooter/TodoFooter.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Filter } from '../../types/Filter';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  setFilter: React.Dispatch<React.SetStateAction<Filter>>;
+  filter: Filter;
+  todos: Todo[];
+};
+
+export const TodoFooter: React.FC<Props> = ({ setFilter, filter, todos }) => {
+  const todosLeft = todos.filter(todo => !todo.completed).length;
+
+  const handeClearCompleted = () => {};
+
+  const returnValue =
+    todos.length > 0 ? (
+      <footer className="todoapp__footer" data-cy="Footer">
+        <span className="todo-count" data-cy="TodosCounter">
+          {`${todosLeft} items left`}
```

Comment:
don't forget to handle case when there is only one item left (you need to remove 's')